src/resolvers/mutation.ts

```diff
@@ -55,7 +54,7 @@ export async function upsertSavedItem(
 
     if (upsertedItem == undefined) {
       console.info(`savedUrl: ${savedItemUpsertInput.url}`);
-      throw new ApolloError(`unable to add an item`);
+      throw new Error(`unable to add an item`);
```

Comment:
Same question, do we want this to be a GraphQLError?

Reply:
Can put this in a follow-up ticket as I was trying to maintain current behavior as much as possible and limit unrelated changes.